files/usr/sbin/jeos-config

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
+# Copyright (c) 2020 SUSE LLC
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+. /etc/os-release
+. "/usr/share/jeos-firstboot/jeos-firstboot-functions"
```

Comment:
Aware of that, but just executed if has the appropriate ${hook} function, anyway, need to improve that, any ideas? Following FHS should be in share, my initial idea was to move the modules to a separate subfolder called modules, what do you think?

Reply:
`jeos-firstboot` and `jeos-config` are very similar anyway, and so it might even make sense to merge them into one file and thus not need `jeos-firstboot-functions`. A separate `modules/` subdirectory would work as well, I don't think we have to be backwards-compatible there.

Reply:
Yes, thought about that as well, kept simple at first, but if we get too much overlap means some more of the code can go in the shared library -functions.